gnovm/docs/go-gno-compatibility.md

```diff
@@ -6,33 +6,57 @@
 
 Legend: full, partial, missing, TBD.
 
-| keyword     | status |
-|-------------|--------|
-| break       | TBD    |
-| case        | TBD    |
-| chan        | TBD    |
-| const       | TBD    |
-| continue    | TBD    |
-| default     | TBD    |
-| defer       | TBD    |
-| else        | TBD    |
-| fallthrough | TBD    |
-| for         | TBD    |
-| func        | TBD    |
-| go          | TBD    |
-| goto        | TBD    |
-| if          | TBD    |
-| import      | TBD    |
-| interface   | TBD    |
-| map         | TBD    |
-| package     | TBD    |
-| range       | TBD    |
-| return      | TBD    |
-| select      | TBD    |
-| struct      | TBD    |
-| switch      | TBD    |
-| type        | TBD    |
-| var         | TBD    |
+| keyword     | support                |
+|-------------|------------------------|
+| break       | full                   |
+| case        | full                   |
+| const       | full                   |
+| continue    | full                    |
+| default     | full                   |
+| defer       | full                   |
+| else        | full                   |
+| fallthrough | full                   |
+| for         | full                   |
+| func        | full                   |
+| go          | missing (after launch) |
+| goto        | full                   |
+| if          | full                   |
+| import      | full                   |
+| interface   | full                    |
+| package     | full                   |
+| range       | full                    |
+| return      | full                   |
+| select      | TBD                    |
+| struct      | full                   |
+| switch      | full                   |
+| type        | full                   |
+| var         | full                   |
+
+## Native types
+
+| type                                          | usage                  | persistency                               | comment                                   |
+|-----------------------------------------------|------------------------|-------------------------------------------|-------------------------------------------|
+| `bool`                                        | full                   | full                                      |                                           |
+| `byte`                                        | full                   | full                                      |                                           |
+| `float32`, `float64`                          | full                   | full                                      | need review of determinism across systems |
+| `int`, `int8`, `int16`, `int32`, `int64`      | full                   | full                                      | potential support of 128, 256 later       |
+| `uint`, `uint8`, `uint16`, `uint32`, `uint64` | full                   | full                                      | potential support of 128, 256 later       |
```

Comment:
link to relevant issues/prs?

Reply:
Undocumented, but we've discussed the possibility of supporting that length, which is a native feature in Solidity. It could aid in the transition, but it's currently just an idea, not a confirmed plan.

Reply:
well, i'd say that we either don't mention it on the doc or we specify that it's still being discussed with a reference to the issue